src/main/java/com/hexagonal/courseregistration/course/application/error/Message.java

```diff
@@ -0,0 +1,15 @@
+package com.hexagonal.courseregistration.course.application.error;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public enum Message {
+  NOT_PROFESSOR("Not Professor"),
+  NOT_EXIST_USER("Not Exist User"),
+  EXIST_COURSE("Exist Course Name"),
+  INVALID_SCORE("Invalid Score"),
+  INVALID_DURATION("Invalid Course Duration");
```

Comment:
```suggestion
  INVALID_DURATION("Invalid Course Duration").
  ;
```

이렇게 작업해두시면 이후에 복붙 또는 삭제 등 enum 값 변경에 편리합니다ㅎㅎ

Reply:
@MangKyu `.`이 아니라 `,` 아닌가요?!